pkg/users/users_test.go

```diff
@@ -204,12 +198,11 @@ func TestUsersCompleteEmailVerification(t *testing.T) {
 	SetAPIKey("test")
 
 	expectedResponse := User{
-		ID:              "user_unmanaged_id",
-		UserType:        Unmanaged,
-		Email:           "marcelina@foo-corp.com",
-		FirstName:       "Marcelina",
-		LastName:        "Davis",
-		EmailVerifiedAt: "2021-07-25T19:07:33.155Z",
+		ID:            "user_unmanaged_id",
```

Comment:
not really a big deal but maybe we should consider making this generic again since we removed the managed/unmanaged concept?

Reply:
good catch